publications-2019.bib

```diff
@@ -500,4 +500,52 @@ @misc{witte2019fast
     primaryClass={math.NA}
 }
 
+@misc{brun2019iterative,
+    title={An iterative staggered scheme for phase field brittle fracture propagation with stabilizing parameters},
+    author={Mats Kirkes{\ae}ther Brun and Thomas Wick and Inga Berre and Jan Martin Nordbotten and Florin Adrian Radu},
+    year={2019},
+    eprint={1903.08717},
+    archivePrefix={arXiv},
+    primaryClass={math.NA}
+}
+
+@misc{endtmayer2019multigoaloriented,
+    title={Multigoal-oriented optimal control problems with nonlinear PDE constraints},
+    author={Bernhard Endtmayer and Ulrich Langer and Ira Neitzel and Winnifried Wollner and Thomas Wick},
```

Comment:
The authors are in a different order here:
https://www.ricam.oeaw.ac.at/files/reports/19/rep19-07.pdf

Reply:
Interesting observation, but that's the order reported by arXiv: https://arxiv.org/abs/1903.02799

Interestingly, when I click on the PDF on arXiv, the order is indeed the one from the link in the PDF you link to. I'll go with your order.

Reply:
Weird. I didn't notice the inconsistency.